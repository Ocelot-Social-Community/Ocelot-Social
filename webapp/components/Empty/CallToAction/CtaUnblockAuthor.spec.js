import { shallowMount } from '@vue/test-utils'

import Component from './CtaUnblockAuthor.vue'

const localVue = global.localVue

describe('CtaUnblockAuthor.vue', () => {
  let propsData, wrapper, mocks

  beforeEach(() => {
    propsData = {
      author: {
        id: 'u-123',
        slug: 'user-123',
        name: 'User 123',
      },
    }
    mocks = {
      $t: jest.fn((t) => t),
    }
  })

  const Wrapper = () => {
    return shallowMount(Component, { propsData, localVue, mocks })
  }

  describe('shallowMount', () => {
    beforeEach(() => {
      wrapper = Wrapper()
    })

    it('renders', () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
