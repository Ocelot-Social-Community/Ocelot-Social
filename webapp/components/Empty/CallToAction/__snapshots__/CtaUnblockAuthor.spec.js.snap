// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CtaUnblockAuthor.vue shallowMount renders 1`] = `
<ds-space-stub marginbottom="large" margin="xxx-small" centered="true" tag="div">
  <ds-space-stub marginbottom="small" tag="div"></ds-space-stub>
  <ds-heading-stub tag="h4">
    contribution.comment.commenting-disabled.blocked-author.reason
  </ds-heading-stub>
  <ds-text-stub tag="p">
    contribution.comment.commenting-disabled.blocked-author.call-to-action
  </ds-text-stub>
  <nuxt-link to="[object Object]">
    <base-button-stub filled="true" icon="arrow-right" size="regular" type="button">
      contribution.comment.commenting-disabled.blocked-author.button-label
    </base-button-stub>
  </nuxt-link>
</ds-space-stub>
`;
